src/renderer/components/+preferences/extensions.tsx

```diff
@@ -11,18 +11,21 @@ import type { RegisteredAppPreference } from "./app-preferences/app-preference-r
 import { ExtensionSettings } from "./extension-settings";
 import { Preferences } from "./preferences";
 import extensionsPreferenceItemsInjectable from "./extension-preference-items.injectable";
+import currentPathParametersInjectable from "../../routes/current-path-parameters.injectable";
+import rendererExtensionsInjectable from "../../../extensions/renderer-extensions.injectable";
 
 interface Dependencies {
   preferenceItems: IComputedValue<RegisteredAppPreference[]>;
+  extensionName: string;
 }
 
-const NonInjectedExtensions = ({ preferenceItems }: Dependencies) => (
+const NonInjectedExtensions = ({ preferenceItems, extensionName }: Dependencies) => (
   <Preferences data-testid="extension-preferences-page">
     <section id="extensions">
-      <h2>Extensions</h2>
-      {preferenceItems.get().map((preferenceItem) => (
+      <h2>{extensionName} settings</h2>
```

Comment:
I think we should use consistent naming (settings vs preferences)

```suggestion
      <h2>{extensionName} preferences</h2>
```